tests/integration/targets/pipx/tasks/main.yml

```diff
@@ -0,0 +1,87 @@
+---
+- name: install pipx
+  pip:
+    name: pipx
+    extra_args: --user
+
+##############################################################################
+- name: install application tox
```

Comment:
Since you make sure that this task is `changed`, you should prepend another task which makes sure that `tox` isn't there.

Reply:
I thought I could count on that given the test is `destructive`, but I adding as a good practice nonetheless

Reply:
Destructive is a flag which prevents tests being run in some environments (like when you don't specify `--docker` when running ansible-test; in that case you have to actively tell ansible-test to still run destructive tests if you want that to happen). Ansible-test does not use it to restrore the system in some way.